### ISSUE TITLE
strange issue while finetuning

### ISSUE THREAD

**Issue by giddyyupp** [karpathy/neuraltalk2#162]
i am training on Turkish captions. and after 125k iterations i enabled finetuning on Flickr30k dataset. but my validation loss increases constantly each iteration. and also my bleu score dropped after finetune. do you think of any reasons for that. 
thanks in advance.

**Comment by hhxxttxsh** [comment]
It is overfitting right now. Is your training loss getting closer to 0? How do you choose the number of iterations before your fine tuning for different dataset?

**Comment by j2233ack** [comment]
How do you train your language captions?
